ui-admin/src/study/surveys/FormOptionsModal.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import { Survey, VersionedForm } from 'api/api'
+import Modal from 'react-bootstrap/Modal'
+import { Button } from 'components/forms/Button'
+import { faArrowRight } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { SaveableFormProps } from './SurveyView'
+import { DocsKey, ZendeskLink } from 'util/zendeskUtils'
+import InfoPopup from 'components/forms/InfoPopup'
+
+/** component for selecting versions of a form */
+export default function FormOptionsModal({
+  workingForm, updateWorkingForm, onDismiss
+}:
+                                          { workingForm: VersionedForm,
+                                            updateWorkingForm: (props: SaveableFormProps) => void,
+                                            onDismiss: () => void}) {
+  return <Modal show={true} onHide={onDismiss} size="lg">
+    <Modal.Header closeButton>
+      <Modal.Title>{workingForm.name} - configuration</Modal.Title>
+    </Modal.Header>
+    <Modal.Body>
+      <FormOptions workingForm={workingForm} updateWorkingForm={updateWorkingForm}/>
+      <div className="fw-light fst-italic mt-4">
+        Note: you must  &quot;Save&quot; the form
+        for changes to these options to take effect.
+      </div>
+    </Modal.Body>
+    <Modal.Footer>
+      <Button variant="secondary" onClick={onDismiss}>
+        Done
+      </Button>
+    </Modal.Footer>
+  </Modal>
+}
+
+/**
+ *
+ */
+export const FormOptions = ({ workingForm, updateWorkingForm }:
+{ workingForm: VersionedForm,
+  updateWorkingForm: (props: SaveableFormProps) => void}) => {
+  const isSurvey = !!(workingForm as Survey).surveyType
```

Comment:
nit: should this check exist in the above modal and display an error if not the right type? it gives some amount of code smell that you can create a `<FormOptions` and it will silently render nothing if provided the 'wrong' kind of form

Reply:
just added a "this form has no configurable options" message so it won't be as mystifying